omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -4,8 +4,9 @@ OMERO.server installation on OS X with Homebrew
 .. topic:: Overview
 
     This walkthrough demonstrates how to install OMERO on a clean Mac
-    OS X system (10.9 or later) using Homebrew.  Note that this
-    demonstrates how to install OMERO.server *from the source code*
+    OS X system (10.9 or later) using Homebrew.  This demonstrates how to install
+    OMERO.server, either from the downloaded OMERO.server.zip
+    or *from the source code*
```

Comment:
Did you mean to mark it up as italics ? Maybe bold would be better (two stars)

Reply:
Don't think it needs bold or italics really.

Reply:
agree, plain is the best